gnomad/utils/transcript_annotation.py

```diff
@@ -331,7 +398,10 @@ def tx_annotate_variants(
         **ht[vep_annotation],
         **tx_ht[ht[vep_annotation].transcript_id, ht[vep_annotation].gene_id],
     )
-    ht = ht.annotate_globals(tissues=tissues)
+    ht = ht.annotate_globals(
+        tissues=tissues,
+        exp_prop_mean_tissues=exp_prop_mean_tissues,
```

Comment:
this will annotate with two of the same list if the user doesn't specify values for `tissues_to_exclude_from_mean`, right? it's a bit extra code, but this could be:
```
if tissues == exp_prop_mean_tissues:
    return ht.annotate_globals(tissues=tissues)
return ht.annotate_globals(
    tissues=tissues,
    exp_prop_mean_tissues=exp_prop_mean_tissues,
)
```

Reply:
I think I like having them both on there so I can just use `exp_prop_mean_tissues` if I want to in downstream use rather than needing to check for it's existence and then use `tissues` if it doesn't exist